variables.tf

```diff
@@ -123,3 +123,8 @@ variable "kubeconfig_name" {
   description = "Override the default name used for items kubeconfig."
   default     = ""
 }
+
+variable "multiple_worker_group_iam_roles" {
+  description = "If provided, an IAM role will be created for each worker group instead of all worker groups sharing the same role."
+  default     = ""
```

Comment:
It looks like this is a list that would need to be the same length as the amount of work groups? Is that correct?

Reply:
No, this was a flag/boolean - it's used here: https://github.com/terraform-aws-modules/terraform-aws-eks/pull/137/files#diff-0973ac2dc1229ea1662319bdc6954f08R161 and here: https://github.com/terraform-aws-modules/terraform-aws-eks/pull/137/files#diff-b66ca10954cbb6be58939e90083e932fR28
Basically, "if not empty make multiple IAM roles"

I'm not positive that this is the best way of accomplishing this, though - it might just be best to optionally include the IAM role as part of the workers group settings.